Server/index.js

```diff
@@ -6,29 +6,31 @@ const middlewares = jsonServer.defaults();
 const loginResponse = require("./response/login_response_200_OK.json");
 const signUpResponse = require("./response/signUp_response_200_OK.json");
 const verificationResponse = require("./response/verification_response_200_OK.json");
+const errorResponse = require("./Response/error_Response_404.json");
 const dbjson = require("./db.json");
 
+
+server.use(middlewares);
+server.use(jsonServer.bodyParser); // Sirve para interpretar el request 
+
 // AUTHENTICATION ENDPOINTS
 
 server.post("/login", (req, res) => {
     const delay = 2000; // In milliseconds
     setTimeout(() => {
-
         console.log("request: " + JSON.stringify(req.body));
 
-        let filteredUsers = dbjson.users.filter(
-            (user) => user.documentNumber == req.body.documentNumber
-        );  
-
-        if (filteredUsers.length === 0) {
-            res.status(404).jsonp({
-                status: "error",
-                data: null,
-                message: "Credentials are incorrect",
-            });
-        } else {
-            res.status(202).jsonp(loginResponse);
+        console.log("El correo es = "+ req.body.email);
+        if (req.body.email == "" ) {
```

Comment:
Agregale una condicion mas que el teto ingresado sea de una formato de correo valido. Link de referencia: https://stackoverflow.com/questions/25471114/how-to-validate-an-e-mail-address-in-swift

Reply:
Sorry me equivoque. Esta bien lo que hiciste, pero me referìa a hacerlo a nivel de aplicacion en el viewModel.